third_party/utils/src/test/io/bazel/rulesscala/utils/TestUtil.scala

```diff
@@ -7,14 +7,13 @@ import scala.reflect.io.AbstractFile
 import scala.reflect.io.Directory
 import scala.reflect.io.PlainDirectory
 import scala.reflect.io.VirtualDirectory
-import scala.tools.cmd.CommandLineParser
 import scala.tools.nsc.CompilerCommand
 import scala.tools.nsc.Global
 import scala.tools.nsc.Settings
 import scala.tools.nsc.reporters.StoreReporter
 import io.bazel.rulesscala.dependencyanalyzer.DependencyTrackingMethod
 
-object TestUtil {
+object TestUtil extends CompilerAPICompat {
```

Comment:
Would it be possible to make `CompilerAPICompat` an object instead of a trait? It's not a big deal I guess, but on the other hand you are adding `tokenize` to the api of `TestUtil` and I'm not sure it belongs here.

Reply:
Done. I was thinking about doing that, too, as I tend to prefer helper objects to mixins for similar reasons.

I also renamed it to `CommandLineParserAdapter` to reflect the very specific reason for its existence.